lisa/operating_system.py

```diff
@@ -357,7 +357,7 @@ def update_packages(
 
     def capture_system_information(self, saved_path: Path) -> None:
         # avoid to involve node, it's ok if some command doesn't exist.
-        self._node.execute("uname -vrio").save_stdout_to_file(saved_path / "uname.txt")
+        self._node.execute("uname -vrmo").save_stdout_to_file(saved_path / "uname.txt")
```

Comment:
Please make sure it well tested on major distros.

Reply:
sure, will check tomorrow.

Reply:
I have tested below images.

![image](https://user-images.githubusercontent.com/10083705/186205809-eeb0a373-0d3a-40b4-89b4-c4a29018cf3d.png)